cleanlab/internal/multilabel_utils.py

```diff
@@ -64,3 +64,34 @@ def get_onehot_num_classes(labels, pred_probs=None):
             "wrong format for labels, should be a list of list[indices], please check the documentation in find_label_issues for further information"
         )
     return y_one, num_classes
+
+
+def int2onehot(labels, K) -> np.ndarray:
```

Comment:
args should have type annotations

Reply:
Added: https://github.com/aditya1503/cleanlab/blob/627b297f286e0d4b0289b3c9f724ad39ae2866a1/cleanlab/internal/multilabel_utils.py#L72